app/app.go

```diff
@@ -326,6 +326,8 @@ func NewEvmos(
 	bApp.SetVersion(version.Version)
 	bApp.SetInterfaceRegistry(interfaceRegistry)
 
+	InitializeEVMConfiguration(bApp.ChainID())
```

Comment:
Instead of panic inside the function, I would. propagate the error up to this point and then return a panic here. This way the panic won't be obscured by the function.